lib/puppet/ssl/host.rb

```diff
@@ -429,6 +429,7 @@ def load_crls(path)
   # @raise [Puppet::Error] if text form of found certificate bundle is invalid
   #                        and cannot be loaded into cert objects
   # @return [Boolean] true if the CA certificate was found, false otherwise
+  # @deprecated Use Puppet::SSL::StateMachine instead.
```

Comment:
just delete the method

Reply:
`Puppet::SSL::Host` calls it so can't delete it.

Reply:
Does https://tickets.puppetlabs.com/browse/PUP-9513 also encompass this deprecation?

Reply:
Ah yeah it should. I'll update the ticket.